tests/integration/test_payment.py

```diff
@@ -0,0 +1,184 @@
+import os
+import string
+import time
+
+import pytest
+
+from jumpscale.loader import j
+
+WALLET_NAME = os.environ.get("WALLET_NAME")
+WALLET_SECRET = os.environ.get("WALLET_SECRET")
+TRANSACTION_FEES = 0.1
+zos = j.sals.zos
+
+
+def info(msg):
+    j.core.logging.info(msg)
+
+
+def get_funded_wallet():
+    if WALLET_NAME and WALLET_SECRET:
+        wallet = j.clients.stellar.get(WALLET_NAME, network="TEST", secret=WALLET_SECRET)
+        return wallet
+    else:
+        raise ValueError("Please provide add Values to the environment variables WALLET_NAME and WALLET_SECRET")
+
+
+def create_new_wallet():
+    wallet_name = j.data.idgenerator.nfromchoices(10, string.ascii_letters)
+    wallet = j.clients.stellar.new(wallet_name, network="TEST")
```

Comment:
a client is created here and other config objects, do we save them? and do we cleanup later?

Reply:
No, it's not saved and they will die once the test is finished.